pkg/validation/labels.go

```diff
@@ -0,0 +1,43 @@
+package validation
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+)
+
+// ValidateLabelsOnCCreate
+// Checks labels on creation
+func ValidateLabelsOnCreate(labels map[string]string) error {
+	if len(labels) == 0 {
+		return errors.New("labels must not be empty")
+	}
+
+	return nil
+}
+
+// ValidateLabelsOnUpdate
+// Checks if the old and new label set are exactly the same
+func ValidateLabelsOnUpdate(oldLabels, newLabels map[string]string) error {
```

Comment:
labels zijn in kubernetes zo ie zo imutable. Dus dit lijkt me overbodig. Zat dit ook in de oude admission controller?

Reply:
laat maar dit lijkt niet zo te zijn. Dit hebben we zelf zo toegevoegd